### PR TITLE
Reset upgrade playbook for 1.6

### PR DESCRIPTION
**Pull request by david-martin**
## Additional Information
Reset the upgrade playbook to remove 1.5 specific upgrade bits.

## Verification Steps
n/a

### CODE REVIEW

File: playbooks/upgrades/upgrade.yaml [hunk 1]
```diff
@@ -116,19 +81,16 @@
     - include_role:
         name: middleware_monitoring
         tasks_from: upgrade/grafana
-      when: target_version.stdout == "release-1.4.1"
```
Comment (david-martin) [inline playbooks/upgrades/upgrade.yaml]: @pb82 I'm removing the extra version checks here and relying on the prerequisite at the start.
Does it make sense to include these monitoring tasks as a starting point for the next upgrade?

**Comment by davidffrench** [comment]
@david-martin Do we want to wait for 1.5 to be released before this is merged?

File: playbooks/upgrades/upgrade.yaml [hunk 2]
```diff
@@ -156,8 +117,5 @@
       vars:
         fuse_image_tag: "{{ upgrade_fuse_image_tag }}"
 
-- import_playbook: install_user_rhsso.yml
-- import_playbook: 3scale_upgrade_2.5_to_2.6.yml
```
Comment (david-martin) [inline playbooks/upgrades/upgrade.yaml]: These 2 are specific to upgrade to 1.5

File: playbooks/upgrades/upgrade.yaml [hunk 3]
```diff
@@ -73,41 +73,6 @@
...
-    - name: Install application metrics
-      include_role:
-        name: application_monitoring
-      when: application_metrics
-
-    - name: Upgrade code ready workspaces 1.0 to 1.2
-      include_role:
-        name: code-ready
-        tasks_from: upgrade_1.0_to_1.2.yml
-
```
Comment (david-martin) [inline playbooks/upgrades/upgrade.yaml]: All of above are specific to upgrade to 1.5.

@wei-lee is there a generic `upgrade` task for any of the mobile components at this time that could stay here on master?

File: roles/3scale/tasks/upgrade.yml [hunk 4]
```diff
@@ -1,226 +0,0 @@
-#upgrade
```
Comment (david-martin) [inline roles/3scale/tasks/upgrade.yml]: Removing 3scale 2.5 specific upgrade file to avoid confusion.

**Comment by david-martin** [comment]
> @david-martin Do we want to wait for 1.5 to be released before this is merged?

I think it's safe to do this as soon as the release branch (v1.5 in this case) is created.
Once the release has it's first RC, any upgrade fixes are only relevant to that release branch, not master.

Does that sound reasonable?

File: docs/release.md [hunk 5]
```diff
@@ -10,6 +10,21 @@
     
     ``` ./scripts/release.sh -b v2.7 -r release-2.7.0-rc1```
 
+
+### Resetting the upgrade playbook on master
```
Comment (david-martin) [inline docs/release.md]: @maleck13 @mikenairn I'd appreciate your review of this doc section.
It's kind of what has been done for the previous 2 releases anyways.

**Comment by davidffrench** [comment]
> > Do we want to wait for 1.5 to be released before this is merged?
> 
> @davidffrench I think it's safe to do this as soon as the release branch (v1.5 in this case) is created.
> Once the release has it's first RC, any upgrade fixes are only relevant to that release branch, not master.
> 
> Does that sound reasonable?

Ya, I think that sounds reasonable. Right now we don't test the cherry picked PR much as this is done off master. We will need to make sure both are tested correctly with this change which would also make sense anyway.

**Comment by david-martin** [comment]
@maleck13 @mikenairn Would appreciate your thoughts on tagged section above

File: docs/release.md (hunk 5 above)
Comment (maleck13) [reply docs/release.md]: looks reasonable to me

Comment (mikenairn) [reply docs/release.md]: Sounds good to me also

**Pull request merged**